boards/nxp/lpcxpresso54114/Kconfig.defconfig

```diff
@@ -0,0 +1,23 @@
+# LPCXpresso54114 board
+
+# Copyright (c) 2017, 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_LPCXPRESSO54114_LPC54114_M4 || BOARD_LPCXPRESSO54114_LPC54114_M0
+
+config BOARD
```

Comment:
Do we need to set `CONFIG_BOARD` here? I believe the build system will set it to `lpcxpresso54114` if we don't set it here.

Reply:
probably not. I think I did this from an earlier template and forgot to clean that out.